tests/storages_tests/journal_tests/test_journal.py

```diff
@@ -67,15 +79,19 @@ def __exit__(
 
 
 @pytest.mark.parametrize("log_storage_type", LOG_STORAGE)
-def test_concurrent_append_logs_for_multi_processes(log_storage_type: str) -> None:
+@pytest.mark.parametrize("deprecated_classname", [True, False])
```

Comment:
I think we should not test the behavior of the deprecated classes. They should be tested only as they are deprecated.

Reply:
Ditto the reply above.